wear/src/main/java/org/gdg/frisbee/android/GdgWatchFace.java

```diff
@@ -99,14 +98,15 @@ public void handleMessage(Message message) {
         Paint mMinuteHandPaint;
         Paint mSecondHandPaint;
         Paint mHourMarkerPaint;
-        Paint mTextPaint;
+        Paint mDateTimePaint;
         Bitmap mBackgroundBitmap;
-        Bitmap mDarkBackgroundBitmap;
-        Bitmap mLightBackgroundBitmap;
         Bitmap mGrayBackgroundBitmap;
+        int backgroundColor;
+        int timeSetting;
         boolean mAmbient;
         boolean mLightMode = false;
         boolean mDisplayDate = true;
+        boolean displayTime = true;
```

Comment:
`displayTime` and `mDisplayDate` together does not look clean. 
Can you make them consistent?